code/controllers/master.dm

```diff
@@ -63,6 +67,12 @@ var/CURRENT_TICKLIMIT = TICK_LIMIT_RUNNING
 		Master = src
 
 /datum/controller/master/Destroy()
+	running = FALSE
+	//Give the loop thread some shutdown time
+	while(looping)
+		sleep(1)
+	for(var/datum/subsystem/ss in subsystems)
+		qdel(ss)
```

Comment:
The mc can, and is intended to get deleted while preserving the subsystems, this has to be moved elsewhere.